priam/src/main/java/com/netflix/priam/tools/CopyInstanceData.java

```diff
@@ -52,10 +58,16 @@ public void run(Bootstrap<?> bootstrap, Namespace namespace) throws Exception {
         }
     }
 
-    private SDBInstanceData getSimpleDB(String domain, String region) {
+    private static SDBInstanceData getSimpleDB(String domain, String region, PriamConfiguration priamConfiguration) {
```

Comment:
Just a thought:  The SDB domain and region are already included in the Priam config.  We could remove them from the command line parameters completely and just extract them from the config.  Alternately we could revert to the old way and just add an optional "SDBAssumeRoleARN" parameter to the commands.  I'm also fine just leaving as is and coming back to this later.